### ISSUE TITLE
Different evaluation results on MOT16 training set

### ISSUE THREAD

**Issue by iriswang02** [Zhongdao/Towards-Realtime-MOT#170]
Hi, @Zhongdao 

Thank you for your code.

I have successfully installed your repo together with essential requirements. It runs very well but I just want to check one specific detail with you. Hope you can give some advice.

The performance results you shared in README.md file is tested on the MOT-16 training set by using Nvidia Titan Xp GPU.

I do exactly what you did but get a slightly different result. For example, using JDE-1088x608, I get 73.1 on MOTA instead of 74.8. I'm not sure whether this small difference comes from the different GPU, may I know your opinion on this? And is it normal?

Thank you for your great help!

**Comment by IwenLeeO** [comment]
Hi @iriswang02 I have some questions when training the model, could you please tell me how to train the model to get 73.1using JDE -1088x608. i always get 54% .Thanks

**Comment by nemonameless** [comment]
@Zhongdao  same question 73.1 on MOTA instead of 74.8

**Comment by Moens007** [comment]
> Hi @iriswang02 I have some questions when training the model, could you please tell me how to train the model to get 73.1using JDE -1088x608. i always get 54% .Thanks

yes,I am always this score and I do not know why, maybe the numbers of GPU or batchsize cause?  i want to know your parameters about it.
